### ISSUE TITLE
Sort storage scanner output

### ISSUE THREAD

**Issue by josephcsible** [McJty/RFTools#1604]
There should be some way of sorting the items in the storage scanner, like you can do now with modular storage.

**Comment by jesmores** [comment]
Yes please! I just built a modular storage + scanner + xnet + wireless tablet system for the first time in E2E and was highly disappointed when I found I could not have the same sorting options in the scanner/tablet as the modular storage blocks did :(
